.coveragerc

```diff
@@ -81,6 +81,10 @@ omit =
     homeassistant/components/aprilaire/climate.py
     homeassistant/components/aprilaire/coordinator.py
     homeassistant/components/aprilaire/entity.py
+    homeassistant/components/apsystems_local/__init__.py
```

Comment:
Final change is to rename the domain to `apsystems`. Otherwise this is good to go!

Reply:
Even though there's the possibility for a cloud integration? What would the domain for the cloud integration be?